SomosMasApp/SomosMasApp/App/Shared/Modules/LogIn/Controller/logInViewController.swift

```diff
@@ -40,6 +40,25 @@ class logInViewController: UIViewController {
                 self.warningLabel.isHidden = true
             }
         }
+        // [OT171-25] Function for API Validation:
+        userValidation()
+    }
+    
+    func userValidation() {
+        let user = Credentials(email: emailTextField?.text, password: passwordTextField?.text)
+        loginStatus(user: user)
+        // text validation
+    }
+    
+    func loginStatus(user: Credentials) {
+        LoginService().login(user: user) { token in
+            // Token Saved
+            print("Token has been saved: \(token)")
```

Comment:
in the Jira Ticket saids the you should save the token in the device( "En caso satisfactorio guardar dentro del dispositivo el "token" devuelto ".) but you are not doing that .    Also please take out any prints for the PR

Reply:
please do this in a secure way if you can

Reply:
*Also Please use a more descriptive name 
* this function should be in the view model